apis/logging/v1/output_types.go

```diff
@@ -46,6 +46,13 @@ type Cloudwatch struct {
 	// +required
 	Region string `json:"region,omitempty"`
 
+	// RetentionInDays defines the expiry time for auto created log group in days.
```

Comment:
Is days the only option available? Is it possible to define something like 72h ?

Reply:
In fluentd docs, it's not specified. https://github.com/fluent-plugins-nursery/fluent-plugin-cloudwatch-logs

But from AWS docs, it seems that only a predefined list of values is allowed. https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_PutRetentionPolicy.html#API_PutRetentionPolicy_RequestSyntax

This also matches AWS to the AWS console. Because it's only possible to select predefined values in the console.